doc/backend/env_settings.md

```diff
@@ -21,6 +21,7 @@ To update the list below you will need to:
 | TRACIM_DEFAULT_LANG                                                       | default_lang                                                   | DEFAULT_LANG                                                       |
 | TRACIM_PREVIEW_CACHE_DIR                                                  | preview_cache_dir                                              | PREVIEW_CACHE_DIR                                                  |
 | TRACIM_AUTH_TYPES                                                         | auth_types                                                     | AUTH_TYPES                                                         |
+| TRACIM_USER__PROFILE__READ_ONLY_FIELDS_XYZ                                | user.profile.read_only_fields.xyz                              | USER__PROFILE__READ_ONLY_FIELDS__XYZ                               |
```

Comment:
I not understand _XYZ here because not same syntax in development.ini.sample
I think it is better to add one parameter for each supported type of auth_type to be more explicit. If not, I think the minimum is to show all supported value in development.ini.sample

What do you think about that @lebouquetin ?

Reply:
Since this is not limited to current auth_types, the list should be updated every time an auth_type is added. If this is not a problem, I will make an exhaustive list

Reply:
@Millefeuille42 the documentation must be explicit and exhaustive

Reply:
@lebouquetin The parameter is documented in the sample `.ini` file. As where the in-depth configuration documentation is AFAIK. Now from what I understand, it is required to list all parameters depending on the auth_type. I will de so

Reply:
Listed relevant auth_types in the table and remove XYZ

Reply:
@Millefeuille42 tracim may be use without a `.ini` file. This may seem like duplication of information but:

- documentation here is targeting all users. This is the reference, the source of truth
- documentation in the `.ini` file is the fastest way to help sysadmins to configure their tracim server.

These are different usages, information may be a bit duplicated but it's required